var/spack/repos/builtin/packages/swig/package.py

```diff
@@ -70,3 +70,41 @@ def create_symlink(self):
         @when(_version)
         def autoreconf(self, spec, prefix):
             which('sh')('./autogen.sh')
+
+
```

Comment:
Removing one of these should fix the style/flake8 error.

Reply:
Aha, I think I was running `spack style` on the wrong branch. Thanks!